iron-overlay-manager.html

```diff
@@ -22,6 +23,8 @@
 
     _backdrops: [],
 
+    _backdropElement: document.createElement('iron-overlay-backdrop'),
```

Comment:
Yeah I can confirm it. If I move the `backdrop` import back into `iron-overlay-behavior` the tests pass